### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by ShivamRai2003**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by nikit19 — COMMENTED** [review]
Please describe in your pull request what you are trying to do

**Comment by ShivamRai2003** [comment]
to complete my task i am doing this so how to merge it now

**Comment by nikit19** [comment]
Please go through the README.md 
https://github.com/fossasia/gci17.fossasia.org/blob/gh-pages/README.md
to see how to complete your task correctly

**Comment by ShivamRai2003** [comment]
please tell me how to merge a pull request please

**Comment by ShivamRai2003** [comment]
please tell

**Review by isuruAb — COMMENTED** [review]
Hello @ShivamRai2003 , could you please explain what you tried to do here. you haven't write permissions so you can't merge but maintainers/mentors can do that.

**Comment by nikit19** [comment]
You first need to complete your task properly. Then it will get merged.
Please go through the README.md to understand how to do that

**Comment by ShivamRai2003** [comment]
only one is left that pull a request and merge it

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
-_site/
-.sass-cache/
-./Gemfile.lock
+- name: Shivam rai
```
Comment (abishekvashok) [inline .gitignore:1]: Invalid unrequired change.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Review by isuruAb — CHANGES_REQUESTED** [review]
don't add your information to the .gitignore file.  Upload correct 240x240 sized one image .

**Pull request closed without merging**